cypress/plugins/docker/index.ts

```diff
@@ -36,12 +36,22 @@ export async function dockerRun(opts: {
     const params = opts.params ?? [];
 
     if (params?.includes("-v") && userInfo.uid >= 0) {
-        // On *nix we run the docker container as our uid:gid otherwise cleaning it up its media_store can be difficult
-        params.push("-u", `${userInfo.uid}:${userInfo.gid}`);
-
         if (await isPodman()) {
-            // keep the user ID if the docker command is actually podman
-            params.push("--userns=keep-id");
+            // Note: this setup is for podman rootless containers.
+
+            // In podman, run as root in the container, so we're the current
```

Comment:
I think this is clearer:

```suggestion
            // In podman, run as root in the container, which maps to the current
```